provisioning/device/build/build.cmd

```diff
@@ -0,0 +1,134 @@
+@REM Copyright (c) Microsoft. All rights reserved.
+@REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+@setlocal EnableExtensions EnableDelayedExpansion
+@echo off
+
+set current-path=%~dp0
+rem // remove trailing slash
+set current-path=%current-path:~0,-1%
+
+set build-root=%current-path%\..\..\..
+rem // resolve to fully qualified path
+for %%i in ("%build-root%") do set build-root=%%~fi
+
+rem ensure nuget.exe exists
```

Comment:
For the new builds we may want to switch to use the dotnet tools instead of VS.
dotnet should perform the nuget tasks as well.

That said, we should merge this to get the folder layout started.